plugin/plugins/mijia/mijia_api/infrastructure/credential_provider.py

```
@@ -0,0 +1,756 @@
+        return datetime.now() + timedelta(days=default_days)
+
+    def __del__(self) -> None:
+        """析构函数，关闭HTTP客户端"""
+        try:
+            self._client.close()
+        except Exception:
+            pass
+        
+    async def get_qrcode_async(self) -> tuple[str, str]:
+        """
+        异步获取二维码登录信息，返回 (qr_url, login_url)
```

Ruff (0.15.7)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)